packages/app/src/components/aside/menu.tsx

```diff
@@ -99,6 +135,21 @@ function isActivePath(router: NextRouter, href: Url) {
   return currentPath === hrefPath;
 }
 
+function isCategoryMenuOpen(router: NextRouter, archivedPaths: ArchivedPath) {
```

Comment:
I would suggest the following name.

```suggestion
function isCurrentRouteArchivedPage(router: NextRouter, archivedPaths: ArchivedPath) {
```

Reply:
The function name was changed!